op-e2e/actions/proofs/holocene_invalid_batch_test.go

```diff
@@ -62,41 +66,56 @@ func Test_ProgramAction_HoloceneInvalidBatch(gt *testing.T) {
 		{
 			name: "valid", blocks: []uint{1, 2, 3},
 			holoceneExpectations: holoceneExpectations{
-				safeHeadPreHolocene: 3, safeHeadHolocene: 3,
+				preHolocene: expectations{safeHead: 3}, holocene: expectations{safeHead: 3},
 			},
 		},
 
 		{
 			name: "invalid-payload", blocks: []uint{1, 2, 3}, blockModifiers: []actionsHelpers.BlockModifier{nil, invalidPayload, nil},
 			useSpanBatch: false,
 			holoceneExpectations: holoceneExpectations{
-				safeHeadPreHolocene: 1, // Invalid signature in block 2 causes an invalid _payload_ in the engine queue. Entire span batch is invalidated.
-				safeHeadHolocene:    2, // We expect the safe head to move to 2 due to creation of an deposit-only block.
+				preHolocene: expectations{safeHead: 1, // Invalid signature in block 2 causes an invalid _payload_ in the engine queue. Entire span batch is invalidated.
+					logs: sequencerOnce("could not process payload attributes"),
+				},
+				holocene: expectations{safeHead: 2, // We expect the safe head to move to 2 due to creation of an deposit-only block.
+					logs: sequencerOnce("could not process payload attributes"),
```

Comment:
could we also want to add an assertion for the specific log message that deposit-only attributes are requested?

Reply:
[a9ed3e8](https://github.com/ethereum-optimism/optimism/pull/12889/commits/a9ed3e8b1f39dfb893b0755abaf5850b36cc1807)